<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="dockNestingEnabled">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QListView" name="listView"/>
    </item>
    <item>
     <widget class="QGroupBox" name="controlsGroupBox">
      <property name="title">
       <string>Controls</string>
      </property>
      <layout class="QHBoxLayout" name="controlsHorizontalLayout" stretch="0">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1,1">
         <item>
          <widget class="QPushButton" name="pauseButton">
           <property name="text">
            <string>Pause</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="startButton">
           <property name="text">
            <string>Start</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="stopButton">
           <property name="text">
            <string>Stop</string>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
      <zorder>horizontalLayoutWidget</zorder>
      <zorder></zorder>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="searchGroupBox">
      <property name="title">
       <string>Search</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="3" column="0" colspan="2">
        <layout class="QHBoxLayout" name="searchOptionsHorizontalLayout">
         <item>
          <widget class="QRadioButton" name="nameRadioButton">
           <property name="text">
            <string>by Name</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="extensionRadioButton">
           <property name="text">
            <string>by Extension</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="dateRadioButton">
           <property name="text">
            <string>by Date</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="sizeRadioButton">
           <property name="text">
            <string>by Size</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="findButton">
         <property name="text">
          <string>Find</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLineEdit" name="searchLineEdit"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QFormLayout" name="statusFormLayout">
      <item row="0" column="1">
       <widget class="QLabel" name="currentDirectoryLabel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Current Diretory:</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
   <zorder>listView</zorder>
   <zorder>horizontalLayoutWidget_2</zorder>
   <zorder>controlsGroupBox</zorder>
   <zorder></zorder>
   <zorder>checkBox</zorder>
   <zorder>nameCheckBox</zorder>
   <zorder>extensionCheckBox</zorder>
   <zorder>dateCheckBox</zorder>
   <zorder></zorder>
   <zorder>label</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
